end_to_end/tpu/llama2/13b/2_test_llama2_13b.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
+# This file is both an integration test that runs once a day on a v4-128 and documentation for how to get started with Llama2-13b. 
+# Please make sure you have run end_to_end/tpu/llama2/13b/1_test_llama2_13b.sh before running commands from this file. 
+
+# The flow of this file is as follows:
+# 1. Run decoding, finetuning of Llama2-13B with the converted checkpoint obtained from end_to_end/tpu/llama2/13b/1_test_llama2_13b.sh. Also, run pretraining of Llama2-13B
+# 2. Convert the scanned checkpoint from step 1 into unscanned checkpoint format and run more efficient decoding.
```

Comment:
Since we have moved the `Convert the scanned checkpoint from step 1 into unscanned checkpoint format ` into the first test script `1_test_<model_name>.sh`
let us update the comment here to say something like `2. Run more efficient decoding with the unscanned checkpoint from end_to_end/tpu/llama2/13b/1_test_llama2_13b.sh`

Could you please update the comment here too?
https://github.com/google/maxtext/blob/main/end_to_end/tpu/llama2/70b/2_test_llama2_70b.sh#L8